### PR TITLE
Safe restart PR builds

### PR DESCRIPTION
**Pull request by CyanDevs**
Signed-off-by: Chris Yan <chrisyan@microsoft.com>

### CODE REVIEW

**Comment by asvrada** [comment]
Sorry but I do not follow how this PR would not fail pipeline build as described? For me the code change is entirely about sending email?

**Comment by CyanDevs** [comment]
@asvrada Jenkins usually splits builds into stages as logical separators between different tasks. We have 4 main stages here: check access, checkout (from scm), determine committers (for emails), and parallel downstream builds. When Jenkins is restarted at the last stage using the "Restart..." button in Blue Ocean, it skips the "Determine committers" stage that would initialize the variable `COMMITTER_EMAILS`. This causes the builds to fail as that variable is used at the end, the post build script, to email the committers about the build result. Hope that helps!

**Review by asvrada — APPROVED** [review]

File: .jenkins/pipelines/manual-pr.Jenkinsfile [hunk 1]
```diff
@@ -130,12 +130,16 @@ pipeline {
     post {
         always {
             script {
-                COMMITER_EMAILS.tokenize('\n').each {
```
Comment (CyanDevs) [inline .jenkins/pipelines/manual-pr.Jenkinsfile:133]: @rs-- Any thoughts on just removing the determine committer step and just email the person that triggered the build (or not at all)?

Comment (rs--) [reply .jenkins/pipelines/manual-pr.Jenkinsfile:133]: I'm alright leaving it out. Personally, I tend to keep an eye on my builds anyways and miss the emails cause they go to my personal email address.

**Review by rs-- — APPROVED** [review]

**Pull request merged**